tkey/src/main/java/com/web3auth/tkey/ThresholdKey/Common/PrivateKey.java

```diff
@@ -30,4 +32,14 @@ public static PrivateKey generate() throws RuntimeError {
         }
         return new PrivateKey(result);
     }
+
+    public String toPublic() throws RuntimeError {
+
+        RuntimeError error = new RuntimeError();
+        String result = jniPrivateToPublic(this.hex, error);
```

Comment:
Result will come back uncompressed, can be converted to keypoint and then get_public_key can be used passing the format parameter.